rocky/assets/js/boefjeSetupRunOn.js

```diff
@@ -0,0 +1,36 @@
+function toggleElements(value) {
+  const intervalNumber = document.querySelector(
+    "label[for='id_interval_number']",
+  ).parentElement;
+  const intervalFrequency = document.querySelector(
+    "label[for='id_interval_frequency']",
```

Comment:
Can we select the labels via a class selector? That way we don't get ID conflicts when adding more than one set of toggleable inputs.